### PR TITLE
osd: add latency for wbthrottle

### PR DESCRIPTION
**Pull request by chuanhong-wang**
Signed-off-by: Chuanhong Wang <chuanhong.wang@163.com>

### CODE REVIEW

File: src/os/filestore/WBThrottle.cc [hunk 1]
```diff
@@ -28,6 +28,7 @@ WBThrottle::WBThrottle(CephContext *cct) :
   b.add_u64(l_wbthrottle_ios_wb, "ios_wb", "Written operations");
   b.add_u64(l_wbthrottle_inodes_dirtied, "inodes_dirtied", "Entries waiting for write");
   b.add_u64(l_wbthrottle_inodes_wb, "inodes_wb", "Written entries");
+  b.add_time_avg(l_wbthrottle_lat, "wait", "throttle latency");
```
Comment (yuyuyu101) [inline src/os/filestore/WBThrottle.cc]: the display name should be "wbthrottle_wait"

File: src/os/filestore/WBThrottle.cc [hunk 2]
```diff
@@ -261,7 +262,16 @@ void WBThrottle::clear_object(const ghobject_t &hoid)
...
 void WBThrottle::throttle()
 {
+  utime_t start = ceph_clock_now();
+  bool waited = false;
   Mutex::Locker l(lock);
-  while (!stopping && need_flush())
+
+  while (!stopping && need_flush()) {
     cond.Wait(lock);
+    waited = true;
```
Comment (yuyuyu101) [inline src/os/filestore/WBThrottle.cc]: you can call tinc here instead of extra `waited`

Comment (chuanhong-wang) [inline src/os/filestore/WBThrottle.cc]: @yuyuyu101 done, please help me review again.

**Comment by gregsfortytwo** [comment]
This looks okay at a glance...
jenkins test this please

**Comment by yuriw** [comment]
@liewegas 

Having trouble merging with our script:

--- found 1 issues tagged with wip-yuri2-testing
--- queried 1 prs
branch wip-yuri2-testing-2018-10-22-2118
--- creating branch wip-yuri2-testing-2018-10-22-2118
Switched to a new branch 'wip-yuri2-testing-2018-10-22-2118'
Traceback (most recent call last):
  File "/home/yuriw/wip_master/src/script/build-integration-branch", line 57, in <module>
    pr['head']['repo']['clone_url'],
TypeError: 'NoneType' object has no attribute '__getitem__'

**Comment by liewegas** [comment]
Looks like the original repo this PR is merging from got deleted...

**Comment by liewegas** [comment]
If you'd like to resurrect this change, please reopen with a new pull request.  Thanks!

**Pull request closed without merging**